### ISSUE TITLE
"Show entire file" not working on renamed files

### ISSUE THREAD

**Issue by mast-eu** [gitextensions/gitextensions#6524]
## Current behaviour
When a file is only renamed (without further changes) and the user selects `Show entire file` the file's content is not shown in the commit form (or in the main forms' diff viewer).

## Expected behaviour
The file's content shall be shown just the same way as if it would contain changes.

## Steps to reproduce

1. Rename any file that is already commited.
2. Stage both files (the removed one and the newly added) in the commit dialog.
3. Click on `Show entire file` .

## Screenshots
![ShowRenamedFileCommit](https://user-images.githubusercontent.com/46861028/57080230-60987380-6cf2-11e9-877b-a7ee8ce1e830.png)

![ShowRenamedFile](https://user-images.githubusercontent.com/46861028/57079579-e7e4e780-6cf0-11e9-90be-e83060ee37c5.png)

## Did this work in previous version of GitExtensions
No. 3.0.x and 2.51.x behave the same.

## Environment
- Git Extensions 3.0.2.5232
- Build 23b6f51905006ccdde8701591df706284b4155dc
- Git 2.21.0.windows.1
- Microsoft Windows NT 10.0.17763.0
- .NET Framework 4.7.3362.0
- DPI 96dpi (no scaling)

**Comment by mstv** [comment]
> Possibly related to #5944?

No, "Show entire file" switches whether only the diffs or all lines are displayed. In the current implementation, the file content is not loaded at all for identical files.
